graphs/adapter-api/src/main/kotlin/com/anaplan/engineering/azuki/graphs/adapter/api/GraphCheckFactory.kt

```diff
@@ -9,4 +9,5 @@ interface GraphCheckFactory : CheckFactory {
     fun hasShortestPath(graphName: String, from: Any, to: Any, shortestPath: List<Any>): Check = UnsupportedCheck
     fun hasCycles(graphName: String, hasCycle: Boolean): Check = UnsupportedCheck
     fun getSimpleCycleCount(graphName: String, count: Long): Check = UnsupportedCheck
+    fun hasPath(graphName: String, from: Any, to: Any, result: Boolean): Check = UnsupportedCheck
```

Comment:
💡 I know that `hasShortestPath` above uses `Any`, and this can work for now, but generally speaking I would prefer it as:
```suggestion
    fun <V> hasPath(graphName: String, from: V, to: V, result: Boolean): Check = UnsupportedCheck
```

Reply:
I've kept this as Any since I couldn't find a neat way to display it or a way to pass the data type.